src/features/cartographie/infrastructure/presentation/directives/leaflet-map-tooltip/leaflet-map-tooltip.directive.ts

```diff
@@ -25,9 +23,9 @@ export class LeafletMapTooltipDirective implements OnDestroy, OnChanges {
 
     this._tooltip = tooltip().setContent(this.content);
     this._canHaveTooltip?.tooltipHolder?.bindTooltip(this._tooltip, {
-      direction: 'top',
-      offset: new Point(0, -MARKER_HEIGHT),
+      direction: 'auto',
       opacity: 1,
+      sticky: true,
```

Comment:
Ajouter `@Input() sticky: boolean = false;`
Remplacer `sticky: true,` par `sticky: this.sticky,`
Vérifier que l'on peut laisser `direction: 'auto', ` sans le conditionner avec `this.sticky` si ça ne pose pas de problème lorsque `this.sticky` vaut `false`